Code/GraphMol/FileParsers/MolFileWriter.cpp

```diff
@@ -595,6 +595,8 @@ void GetMolFileAtomProperties(const Atom *atom, const Conformer *conf,
   }
 }
 
+const double MAX_V2000_COORD = 1000000.;
+const double MIN_V2000_COORD = -100000.;
```

Comment:
```suggestion
constexpr double MAX_V2000_COORD = 1000000.;
constexpr double MIN_V2000_COORD = -100000.;
```

Reply:
I also wonder if these values are correct.
The CTAB spec says that the format is:
`xxxxx.xxxx`
I think that means that the ranges are:
```
constexpr double MAX_V2000_COORD = 100000.;
constexpr double MIN_V2000_COORD = -10000.;
```

Reply:
These are the first bad values, everything below or above these should work.  I.e. 99999.9999

Reply:
MAX coord is a bit misleading I agree

Reply:
> These are the first bad values, everything below or above these should work.  I.e. 99999.9999

I don't believe that this is true. I think you have an extra digit

Reply:
> > These are the first bad values, everything below or above these should work.  I.e. 99999.9999
> 
> I don't believe that this is true. I think you have an extra digit 

You only get 5 digits before the decimal point